variables.tf

```diff
@@ -0,0 +1,142 @@
+# Basic Variables
+variable "profile" {
+  default = "the AWS profile to use"
+}
+
+variable "vpc_id" {
+  description = "ID of the VPC the security_group should be connected to"
+}
+
+variable "description" {
+  description = "The description of the security group"
+  default = "Basic http Security-Group"
```

Comment:
This default value seems to be a little out of context

Reply:
I removed it entierly